lib/widgets/layouts/adaptive_layout/adaptive_scaffold_primary_navigation.dart

```diff
@@ -2,24 +2,25 @@ import 'package:fluffychat/pages/chat_list/client_chooser_button.dart';
 import 'package:fluffychat/pages/chat_list/client_chooser_button_style.dart';
 import 'package:fluffychat/widgets/avatar/avatar.dart';
 import 'package:fluffychat/widgets/layouts/adaptive_layout/adaptive_scaffold_primary_navigation_style.dart';
-import 'package:fluffychat/widgets/layouts/adaptive_layout/adaptive_scaffold_shell.dart';
 import 'package:fluffychat/widgets/matrix.dart';
 import 'package:flutter/material.dart';
 import 'package:matrix/matrix.dart';
 import 'package:flutter_gen/gen_l10n/l10n.dart';
 
 class AdaptiveScaffoldPrimaryNavigation extends StatelessWidget {
-  final AdaptiveScaffoldShellController controller;
+  final Future<Profile?>? futureProfile;
```

Comment:
it is current profile? or profile of who?

Reply:
Fixed